src/Transport/Snmp.php

```diff
@@ -7,12 +7,16 @@
 interface Snmp
 {
     /**
+     * Strips $oid prefix from the result keys
+     *
      * @return iterable<string, mixed>
      */
     public function walk(string $oid) : iterable;
 
     /**
+     * Returns complete OID in the result keys
```

Comment:
```suggestion
```

Reply:
You're actually documenting return value here so it's more suitable as suggested. Probably tools will prefer this as well :)

```php
    /**
     * Finds an object by its identifier.
     *
     * This is just a convenient shortcut for getRepository($className)->find($id).
     *
     * @param string $className The class name of the object to find.
     * @param mixed  $id        The identity of the object to find.
     *
     * @return object|null The found object.
     */
```

![image](https://user-images.githubusercontent.com/327717/59762378-6f81b800-9297-11e9-9285-2192a82ab6ad.png)

Reply:
alright